### DISCUSSION TITLE
Account Suspended, Support Unhelpful

### DISCUSSION THREAD

**Discussion by ghost** [community/community#128186] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

A few days ago, my GitHub account (@aarvinr) was randomly suspended. When I reached out to support (ticket 2832534), I was told it was banned due to misuse of GitHub actions. 

I only used GitHub actions for testing and release automation, which are obviously within GitHub's terms, but even after reaching out twice more (tickets 2833048 and 2833510), I was just told the same thing. 

I believe the reason this was caused is that I was using GitHub codespaces for testing some projects, and ended up sending a large amount of HTTP requests, which I'm guessing flagged my account (similar occurrence [here](https://github.com/orgs/community/discussions/48787#top)).

Could someone please help reinstate my account? Like I said before, support just repeatedly sent me the same email telling me the account was closed because of misuse of GitHub Actions.

**Comment by ebndev** [comment, marked as answer]
Hi @mozaralle,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
